### ISSUE TITLE
Does Success Criteria 3.2.6 fail if the relative order of multiple help mechanisms is not consistent across of set of web pages?

### ISSUE THREAD

**Issue by adithyasai16** [w3c/wcag#3668]
I am seeking clarification on a specific aspect related to Success Criteria 3.2.6 of the Consistent Help guidelines. This question pertains to scenarios where a set of web pages incorporates multiple help mechanisms.

If the set of web pages contain multiple help mechanisms, do we take these multiple help mechanisms as a single help entity and look for the relative order before and after the help mechanisms?

Or 

Do we even check relative order within these multiple help mechanisms on how consistent they are present across set of web pages?

Example 1:
I have set of help links after Pricing link and before Products link in header region of the page, now if these multiple help links are not in relative order across set of pages, do we fail this success criteria?
Page 1: Relative order: Products | Help | Support | FAQ | Products
Page 2: Relative order: Products | Help | FAQ | Support | Products

![Example 1 image](https://github.com/w3c/wcag/assets/101878599/1491c9c5-5b23-4615-b2c4-9657ada3d3a5)

Example 2:
I have set of help sub menu items that appear after expanding "help" menu button. The "help" menu button is placed between search button and More options button. Now if these multiple help submenu items are not in relative order across set of pages, do we fail this success criteria?
Page 1 Relative order: Search >> Browse documentations | View architecture guides | View help FAQs | Start a tutorial | ... | >> More options.
Page 2 Relative order:  Search >> Browse documentations | View help FAQs | Start a tutorial | View architecture guides | ... | >>More options.

![Example 2 image](https://github.com/w3c/wcag/assets/101878599/4f326fd3-d695-4022-a3ee-85569913601a)

In conclusion, clarity on the treatment of multiple help mechanisms within the context of Success Criteria 3.2.6 is crucial for maintaining consistency across web pages. Your insights and expertise are highly valued.

**Comment by JAWS-test** [comment]
According to  [SC 3.2.3: Consistent Navigation (Level AA)](https://www.w3.org/WAI/WCAG22/Understanding/consistent-navigation.html) the order should be consistent

**Comment by adithyasai16** [comment]
@JAWS-test does that mean it fails both SC 3.2.3 and 3.2.6 ?

**Comment by JAWS-test** [comment]
> @JAWS-test does that mean it fails both SC 3.2.3 and 3.2.6 in the above scenario?

No, I think it fails only 3.2.3.

It does not fail 3.2.6 as the SC does not consider the order within the help, but only the help in relation to the rest of the page:

> "they occur in the same order relative to other page content" (https://www.w3.org/WAI/WCAG22/Understanding/consistent-help)
